dsa-solutions/lc-solutions/0200-0299/0211-Design Add and Search Words Data Structure.md

```diff
@@ -0,0 +1,197 @@
+---
+id: Design Add and Search Words Data Structure
```

Comment:
replace `id: Design Add and Search Words Data Structure` to `id: design-add-and-search-words-data-structure`